### DISCUSSION TITLE
What is CHNOPS?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#968] [Galaxy Trivia] [answered]
**Bonus:** What is the mass % in 🌿☘️ and 🐀🦥?

**Comment by peterbaccaro** [comment, marked as answer]
CHNOPS is an **acronym that represents the six most abundant elements found in living organisms: carbon (C), hydrogen (H), nitrogen (N), oxygen (O), phosphorus (P), and sulfur (S)**. These elements are essential building blocks of biological molecules, including carbohydrates, lipids, proteins, and nucleic acids, which are crucial for life processes.

The mass percentages of various elements in plants and mammals can vary depending on the specific species, tissues, and physiological conditions. Here is some general information regarding the typical composition of plants and mammals:

**Plants**:
- Carbon (C): Plants are primarily composed of carbon, which forms the backbone of organic molecules. Carbon can constitute around 45% to 50% of the dry weight of plants, although this can vary.
- Oxygen (O): Oxygen is another major component of plants, primarily found in the form of water and carbohydrates. It can account for - approximately 40% to 45% of the dry weight of plants.
- Hydrogen (H): Hydrogen is present in plants as part of water and organic compounds. It generally makes up around 6% to 7% of the dry weight of plants.
- Nitrogen (N): Nitrogen is an essential component of proteins, nucleic acids, and other organic molecules in plants. Its mass percentage can range from 1% to 6% or more, depending on the plant species and tissue type.
- Other elements: Plants also contain smaller amounts of elements like phosphorus (P), sulfur (S), potassium (K), calcium (Ca), magnesium (Mg), and trace elements like iron (Fe), zinc (Zn), and copper (Cu).

**Mammals**:
- Carbon (C): Carbon is the primary constituent of organic matter in mammals. It can account for roughly 18% to 23% of the total body mass.
- Oxygen (O): Oxygen is abundant in mammals, primarily as a component of water and organic molecules. It generally makes up about 60% to 65% of the total body mass.
- Hydrogen (H): Hydrogen is present in mammals as part of water and organic compounds. It typically constitutes around 9% to 11% of the total body mass.
- Nitrogen (N): Nitrogen is a fundamental element in proteins, nucleic acids, and other biomolecules in mammals. Its mass percentage can vary, but it typically ranges from 2% to 4% of the total body mass.
- Other elements: Mammals also contain smaller amounts of elements such as phosphorus (P), sulfur (S), calcium (Ca), potassium (K), sodium (Na), magnesium (Mg), and trace elements like iron (Fe), zinc (Zn), and copper (Cu).
